app/src/main/java/com/example/firstapp/database/dao/MyBuildDao.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.firstapp.database.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import com.example.firstapp.model.MyBuild
+
+@Dao
+interface MyBuildDao {
+    @Query("SELECT * FROM mybuild")
+    fun selectAll(): List<MyBuild>
+
+    @Query("SELECT * FROM mybuild WHERE champion = :champion ORDER BY id ASC")
+    fun selectByChampionName(champion: String): List<MyBuild>
+
+    @Insert
+    fun insert(myBuild: MyBuild)
+
+    @Query("DELETE FROM mybuild WHERE id = :position")
+    fun delete(position: Int)
```

Comment:
position 보다는 그냥 id 로 하는게 좋을 것 같네요
```suggestion
    @Query("DELETE FROM mybuild WHERE id = :id")
    fun delete(id: Int)
```